sdk/textanalytics/Azure.AI.TextAnalytics/tests/DetectLanguageTests.cs

```diff
@@ -249,5 +232,50 @@ public async Task DetectLanguageBatchWithNullTextTest()
             InvalidOperationException ex = Assert.Throws<InvalidOperationException>(() => results[0].PrimaryLanguage.GetType());
             Assert.AreEqual(exceptionMessage, ex.Message);
         }
+
+        private void ValidateInDocumenResult(DetectedLanguage language)
+        {
+            Assert.That(language.Name, Is.Not.Null.Or.Not.Empty);
+            Assert.That(language.Iso6391Name, Is.Not.Null.Or.Not.Empty);
+            Assert.GreaterOrEqual(language.ConfidenceScore, 0.0);
+        }
+
+        private void ValidateBatchDocumentsResult(DetectLanguageResultCollection results, bool includeStatistics = default)
+        {
+            Assert.IsNotEmpty(results.ModelVersion);
```

Comment:
Should we null-check here as well?

Reply:
It is required on the swagger so codegen should help us there.